packages/logic-utils/src/utils/xml-node-to-ui5-node.ts

```diff
@@ -54,3 +56,29 @@ export function getUI5PropertyByXMLAttributeKey(
   const ui5Property = find(properties, ["name", attribute.key]);
   return ui5Property;
 }
+
+export function getUI5NodeFromXMLElementNamespace(
```

Comment:
should this be a **general** logic util or is it too **specific** for a single use case?

Reply:
I think we will use it for other logic too, like in hover (on xmlns attribute value or element/attribute namespace prefix if we want to implement that)

Reply:
Only if this hierarchy is actually valid in api.json